Add unit test coverage for db/repos/rounds

Parent: #1484

### Expected behavior
We need 100% coverage for db/repos/rounds

### Actual behavior
Currently there is no coverage for this repo.

I've just notice an error on this new test file:
```
        1) should return unique round numbers

 1) db
       RoundsRepository
         getMemRounds()
           should return unique round numbers:

      AssertionError: expected [ '2', '1' ] to deeply equal [ '1', '2' ]
      + expected - actual

       [
      +  "1"
         "2"
      -  "1"
       ]
      
    at Context.<anonymous> (test/unit/db/repos/rounds.js:105:52)
    at undefined.next (native)
    at onFulfilled (node_modules/co/index.js:65:19)
    at tryCatcher (node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues (node_modules/bluebird/js/release/async.js:17:14)

```

--- a/test/unit/db/repos/rounds.js
+++ b/test/unit/db/repos/rounds.js
@@ -102,7 +102,10 @@ describe('db', () => {
 				expect(result1).to.have.lengthOf(3);
 				expect(result2).to.have.lengthOf(2);
 				expect(result2[0]).to.have.all.keys('round');
-				return expect(result2.map(r => r.round)).to.be.eql(['1', '2']);
+				return expect(result2.map(r => r.round)).to.have.all.members([
+					'1',
+					'2',
+				]);
 			});
 		});
 

